script/ci

```diff
@@ -10,11 +10,12 @@ export GIT_BRANCH="$HEROKU_TEST_RUN_BRANCH"
 export GIT_COMMIT_SHA="$HEROKU_TEST_RUN_COMMIT_VERSION"
 export GIT_COMMITTED_AT="$(date +%s)"
 
+# run the ruby test suite
+CAPYBARA_DRIVER=headless_chrome bundle exec rake spec
+CAPYBARA_DRIVER=headless_firefox bundle exec rake spec
```

Comment:
Maybe only run the non-feature specs once. Even though they are fast.
```
bundle exec rspec --tag ~type:feature
CAPYBARA_DRIVER=headless_chrome bundle exec rspec --tag type:feature
CAPYBARA_DRIVER=headless_firefox bundle exec rspec --tag type:feature
```

Reply:
Good point with code coverage. This is very much work in progress, and I just wanted to play around with Heroku CI and buildpacks. I'll have to get back to this next week.